apps/charterafrica/src/payload/blocks/BlogContent.js

```diff
@@ -0,0 +1,14 @@
+import blockFields from "../fields/blockFields";
+import richText from "../fields/richText";
+
+const BlogContent = {
+  slug: "blogContent",
+  fields: [
+    blockFields({
+      name: "blogContentFields",
+      fields: [richText({ name: "content", admin: { elements: ["ul"] } })],
```

Comment:
This richText should include pretty much everything, headers, links, etc. Not sure about images if we have a separate block for that.